### ISSUE TITLE
a11y: RangeSlider - inappropriate role for element

### ISSUE THREAD

**Issue by PeterCiuffetti** [appbaseio/reactivesearch#1555]
**Affected Projects**
React

**Library Version:** 3.12.7

**Describe the bug**
RangeSlider uses a button element for the slider.  This also uses aria attribute for accessibility.  The problem is that role="slider" is not allowed on a button element.  [ARIA in HTML specs](https://www.w3.org/TR/html-aria/#example-1) indicate that a button has an implicit role of button and cannot specify another role.

**To Reproduce**
Install the Chrome browser plugin from Deque called Axe
https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd?hl=en

On any page that implements RangeSlider, visit with Chrome. Open the debugging console and move to the Axe tab. Press 'Analyze'.

**Expected behavior**
To fix this issue, range slider is going to have to use a styled div or some other element that allows role=slider

**Screenshots**
![image](https://user-images.githubusercontent.com/1168204/97357914-de4e2880-1870-11eb-96a9-f13ebb3a24b6.png)

**Desktop (please complete the following information):**
 - OS: Mac OSX Catalina
 - Browser chrome
 - Version 86.0.4240.111

**Additional context**
We originally found this issue on the following page in our system:
https://mindscapecommons.net/search?q=%22vr%22
